components/notifications/notifications.tsx

```diff
@@ -101,35 +101,35 @@ export default function Notifications({ token }: { token: string }) {
     }
   }, [fetchNextPage, inView]);
 
-  const handleNotification = (event: MessageEvent) => {
-    const data = JSON.parse(event.data);
-
-    if (data.redirectionLink.includes("meeting")) {
-      toast.info(data.title, {
-        description: data.body,
-        action: {
-          label: "Join",
-          onClick: () => router.push(data.redirectionLink),
-        },
-        duration: 5000,
-      });
-    } else if (data.redirectionLink.includes("room")) {
-      toast.info(data.title, {
-        description: "We will redirect you shortly!",
-        duration: 5000,
-      });
-
-      if (pathName.includes("meeting")) router.push(data.redirectionLink);
+
+  const matchLabelToLink = (link: string) => {
+    if (link.includes("meeting")) {
+      return "Join"
     } else {
-      toast.info(data.title, {
-        description: data.body,
-        action: {
-          label: "View",
-          onClick: () => router.push(`/dashboard/invitations`),
-        },
-        duration: 5000,
-      });
+      return "View"
+    }
+  }
+
+  const handleNotification = (event: MessageEvent) => {
+    const data = JSON.parse(event.data) as Notification;
```

Comment:
```suggestion
    const data = JSON.parse(event.data) satisfies Notification;
```

it's safer to use satisfies there i think

i dont know how it works with data sent by class-service, maybe forced type casting is valid here